lib/wolfe/cleanup.rb

```diff
@@ -65,10 +65,13 @@ def cleanup( config )
       end
 
       def calculate_monthly_date_for_backup_to_keep(one_per_month_timespan)
-        if Date.today - eval(one_per_month_timespan) == Date.today
+        timespan_digit = one_per_month_timespan.split('.')[0].to_i
+        timespan_word = one_per_month_timespan.split('.')[1].to_sym
```

Comment:
I wasn't sure about the naming of the variables. If you have any suggestions, I will use them immediately.